### DISCUSSION TITLE
Welcome to monhttp Discussions!

### DISCUSSION THREAD

**Discussion by koloo91** [koloo91/monhttp#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nanu-c** [comment]
Hey koloo91,
i stumbled upon monhttp and it looks promising in general, i would like to know how can i add tasks? only with the database at the moment?

**Reply by koloo91** [reply]
Hey @nanu-c,
what do you mean by task? Currently, this is only possible via PostgreSQL, but more databases will be added soon.

**Reply by nanu-c** [reply]
What I mean is how can i add a service that is checked? I could add support for notifications via signal messenger :)

**Reply by koloo91** [reply]
You click on the "Service" tab and after that on the "Create" button. Enter the required data like "Service Name" and "Endpoint" and click on the save button at the bottom of the page afterwards.
